delivery_purchase_multi_destination/models/purchase_order.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2025 Tecnativa - Carlos Roca
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class PurchaseOrder(models.Model):
+    _inherit = "purchase.order"
+
+    @api.depends("partner_id")
```

Comment:
Add a explanation comment:

```suggestion
    # Introduce this extra dependency in the compute
    @api.depends("partner_id")
```